### PR TITLE
Fix multiple code bugs

### PR DESCRIPTION
**Pull request by mbolka**
Resolves some bugs that existed in code.
All bugs were reported by SonaryQube tool.
## Additional info 
Please check carefully change that was done on  bundles/EcommerceFrameworkBundle/VoucherService/Token/Dao.php
models/Document/Tag/Video.php

### CODE REVIEW

File: bundles/EcommerceFrameworkBundle/VoucherService/Token/Dao.php [hunk 1]
```diff
@@ -83,12 +83,8 @@ public static function isUsedToken($token, $usages = 1)
...
 
         try {
-            $usages['usages'] = $db->fetchOne($query, $params);
-            if ($usages > $usages) {
-                return $usages['seriesId'];
-            } else {
-                return false;
-            }
+            $result = $db->fetchOne($query, $params);
+            return $result['usages'] >= $usages;
```
Comment (NiklasBr) [inline bundles/EcommerceFrameworkBundle/VoucherService/Token/Dao.php]: You replaced `>` with `>=`, was that intentional and why?

**Comment by brusch** [comment]
Reverted changes in `bundles/EcommerceFrameworkBundle/VoucherService/Token/Dao.php` and fixed bug in `models/Document/Tag/Video.php` - all the other look good so far - thanks!

**Comment by mbolka** [comment]
@brunch Are you sure this Token behavior is correct? This criteria will never be met

**Comment by brusch** [comment]
@mbolka just removed it from this PR, but of course this needs to be further investigated

**Comment by brusch** [comment]
Voucher token fixed in af3b5ea8679b49088b420a09895f8297326929c3

**Comment by mbolka** [comment]
@brusch please check my comments on your commit

**Pull request merged**